Source/Core/DolphinQt/Config/VerifyWidget.cpp

```diff
@@ -51,6 +51,7 @@ void VerifyWidget::CreateWidgets()
   m_summary_text->setReadOnly(true);
 
   m_hash_layout = new QFormLayout(this);
+  m_hash_layout->setFieldGrowthPolicy(QFormLayout::AllNonFixedFieldsGrow);
```

Comment:
I think you should add a comment stating this is needed on macOS so someone doesn't accidentally remove it. (assuming this does fix the issue)

Reply:
Added.